src/main/scala/is/hail/expr/FunctionRegistry.scala

```diff
@@ -1225,6 +1225,25 @@ object FunctionRegistry {
     ``fet`` is identical to the version implemented in `R <https://stat.ethz.ch/R-manual/R-devel/library/stats/html/fisher.test.html>`_ with default parameters (two-sided, alpha = 0.05, null hypothesis that the odds ratio equals 1).
     """,
     "a" -> "value for cell 1", "b" -> "value for cell 2", "c" -> "value for cell 3", "d" -> "value for cell 4")
+
+  register("binomTest", { (nSuccess: Int, n: Int, p: Double, alternative: String) => binomTest(nSuccess, n, p, alternative)
+  },
+    """
+    Performs a binomial test and returns the p-value.
+
+    **Examples**
+
+    Test for allele balance in heterozygote samples under the hypothesis that each allele has equal probablilty of sampling.
```

Comment:
"Test each variant for allele balance across all heterozygous genotypes, under the null hypothesis that the two alleles are sampled with equal probability."